src/rosdiscover/ros2_translator/arch.py

```diff
@@ -0,0 +1,41 @@
+import yaml
+from launch.actions import IncludeLaunchDescription
+from launch_ros.actions import Node
+from launch.launch_description_sources import get_launch_description_from_python_launch_file
+
+description = get_launch_description_from_python_launch_file('robot.launch.py')
+nodes = description.entities
+
+
+def toyaml(nodes):
+    to_yaml = {}
+    no_name = 0
+    for node in nodes:
+        if isinstance(node, IncludeLaunchDescription):
+            description1 = node._IncludeLaunchDescription__launch_description_source.try_get_launch_description_without_context()
+            if description1 is None:
```

Comment:
This if-condition can't be reached. If `node` is `None`, then `isinstance(node, IncludeLaunchDescription)` will evaluate to `False`.